google_monitoring/variables.tf

```diff
@@ -14,6 +14,7 @@ variable "uptime_checks" {
     timeout             = optional(string, "60s")
     period              = optional(string, "300s")
     user_labels         = optional(map(string), {})
+    selected_regions    = optional(list(string), [])
```

Comment:
I think we should add defaults here. I've been adding this to the refractr monitoring recently, google uptime checks require at least 3 regions to work, so I think adding those as defaults would be a good idea, e.g.:

```
selected_regions = optional(list(string), ["EUROPE", "US_OREGON", "US_VIRGINIA"])
```

Also, looking at future costs, 3 regions means 3 check executions, leaving this with the default means 6 check executions, so also a good idea to keep this low in defaults :thinking: